### PR TITLE
Maxbin2 - removing whitespaces from reads filename to fix bug

### PR DESCRIPTION
**Pull request by Ralwoss**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/maxbin2/maxbin2.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         </xml>
     </macros>
     <xrefs>
-        <xref type="bio.tools">maxbin</xref>
+        <xref type="bio.tools">masigpro</xref>
```
Comment (bgruening) [inline tools/maxbin2/maxbin2.xml:20]: that looks wrong ...

File: tools/maxbin2/maxbin2.xml [hunk 2]
```diff
@@ -112,7 +112,7 @@ run_MaxBin.pl
                 <conditional name="inputs">
                     <expand macro="input_type"/>
                     <when value="reads">
-                        <param argument="-reads" type="data" format="fasta,fastq,fastq.gz,fasta.gz" label="Reads file"/>
```
Comment (bgruening) [inline tools/maxbin2/maxbin2.xml:115]: why removing fasta.gz? Does that not work?

File: tools/maxbin2/maxbin2.xml (hunk 1 above)
Comment (Ralwoss) [reply tools/maxbin2/maxbin2.xml]: that occured somehow through merging, fixed it now

File: tools/maxbin2/maxbin2.xml (hunk 2 above)
Comment (Ralwoss) [reply tools/maxbin2/maxbin2.xml]: also fixed

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @Ralwoss

**Pull request merged**